### PR TITLE
completing main entrypoint fixes

### LINKED ISSUE #1: Create entrypoint to project
- [x] create run.py file
- [x] add shebang to script
- [x] make script executable
- [x] create execution entrypoint

### PR DESCRIPTION
**Pull request by imndaiga**
close #1

### CODE REVIEW

**Review by DanSanz — APPROVED** [review]

**Review by Tabixe — APPROVED** [review]
solid work mate!

**Review by Ioana-P — APPROVED** [review]

File: run.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python3
+
```
Comment (Tabixe) [inline run.py:2]: I love this line

**Review by davidjohnbaker1 — COMMENTED** [review]
This looks like beautiful code. Great work, Wachira!

File: run.py (hunk 1 above)
Comment (imndaiga) [reply run.py:2]: thanks

**Review by davidjohnbaker1 — APPROVED** [review]

**Pull request merged**